packages/theme-check-common/src/checks/hardcoded-routes/index.spec.ts

```diff
@@ -0,0 +1,102 @@
+import { expect, describe, it } from 'vitest';
+import { HardcodedRoutes } from './index';
+import { runLiquidCheck, highlightedOffenses } from '../../test';
+
+describe('Module: HardcodedRoutes', () => {
+  it('should report offenses for hardcoded routes', async () => {
+    const sourceCode = `
+      <a href="/"></a>
+      <form action="/search"></form>
+      <a href="/account/login"></a>
+      <a href="/collections/{{ collection.handle }}"></a>
+      <a href="/cart/change?line={{ forloop.index }}&amp;quantity=0"></a>
```

Comment:
I found a use case that we should probably cover:

```liquid
{{ 'Cart' | link_to: '/cart' }}
```

I'm not sure how much the `link_to` filter is used but worth covering! Feel free to address this in a follow up PR.